internal/state/configuration_test.go

```diff
@@ -1276,3 +1285,192 @@ func TestBuildBackendGroups(t *testing.T) {
 		t.Errorf("buildBackendGroups() mismatch: %+v", diff)
 	}
 }
+
+func TestBuildWarnings(t *testing.T) {
+	createBackendRefs := func(names ...string) []BackendRef {
+		backends := make([]BackendRef, len(names))
+		for idx, name := range names {
+			backends[idx] = BackendRef{Name: name}
+		}
+
+		return backends
+	}
+
+	createBackendGroup := func(sourceName string, backends []BackendRef, errMsgs ...string) BackendGroup {
+		return BackendGroup{
+			Source:   types.NamespacedName{Namespace: "test", Name: sourceName},
+			Backends: backends,
+			Errors:   errMsgs,
+		}
+	}
+
+	hr1Rule0 := createBackendGroup(
```

Comment:
is `hr1Rule0` a suitable name? In the other tests in this file backend groups names like `	hr5BackendGroup`.

Reply:
Changed these to `hrXBackendGroupY`